powerline/lemonbar.py

```diff
@@ -0,0 +1,23 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+from powerline import Powerline
+from powerline.lib.dict import mergedicts
+
+
+class LemonbarPowerline(Powerline):
```

Comment:
@S0lll0s I do not know. If lemonbar expects locale encoding in the input you should live this as-is. If lemonbar expects UTF-8 only you should define this method to force UTF-8.